### ISSUE TITLE
Autocorrect is available, but not applied for `Style/ComparableClamp` and `Style/GuardClause` cops

### ISSUE THREAD

**Issue by ydakuka** [rubocop/rubocop#14682]
## Actual behavior

When I run:
```console
rubocop --auto-gen-config
```

rubocop generates a `.rubocop_todo.yml` file that contains:

```yaml
# Offense count: 1
# This cop supports safe autocorrection (--autocorrect).
Style/ComparableClamp:
  Exclude:
    - 'app/presenters/profile_presenter.rb'

# Offense count: 1
# This cop supports safe autocorrection (--autocorrect).
# Configuration parameters: MinBodyLength, AllowConsecutiveConditionals.
Style/GuardClause:
  Exclude:
    - 'app/controllers/profiles_controller.rb'
```

However, when I run rubocop on the individual files, I get the following messages, but there is no note about autocorrection being available:

```console
$ rubocop app/presenters/profile_presenter.rb
Inspecting 1 file
C

Offenses:

app/presenters/profile_presenter.rb:68:23: C: Style/ComparableClamp: Use Comparable#clamp instead.
    @captcha_rate ||= [ ...
                      ^

1 file inspected, 1 offense detected
```
```console
$ rubocop app/controllers/profiles_controller.rb
Inspecting 1 file
C

Offenses:

app/controllers/profiles_controller.rb:71:9: C: Style/GuardClause: Use a guard clause (redirect_to edit_profile_path(@profile) and return unless policy(@profile).update_admin_info?) instead of wrapping the code inside a conditional expression.
        if policy(@profile).update_admin_info?
        ^^

1 file inspected, 1 offense detected
```

I also verified that `rubocop -A` does not perform autocorrection for these offenses.

## Rubocop version

```console
$ rubocop -V
1.81.7 (using Parser 3.3.9.0, rubocop-ast 1.48.0, analyzing as Ruby 3.2, running on ruby 3.2.9) [x86_64-linux]
  - rubocop-capybara 2.22.1
  - rubocop-factory_bot 2.28.0
  - rubocop-migration 0.7.1
  - rubocop-performance 1.26.1
  - rubocop-rails 2.34.0
  - rubocop-rake 0.7.1
  - rubocop-rspec 3.8.0
  - rubocop-rspec_rails 2.32.0
  - rubocop-thread_safety 0.7.3
```

**Comment by ydakuka** [comment]
`Style/ComparableClamp`

> This cop supports autocorrection for ‘if/elsif/else` bad style only. Because `ArgumentError` occurs if the minimum and maximum of `clamp` arguments are reversed. When these are variables, it is not possible to determine which is the minimum and maximum...

`Style/GuardClause`

> NOTE: Autocorrect works in most cases except with if-else statements
